whisper/whisperv6/config.go

```diff
@@ -18,12 +18,14 @@ package whisperv6
 
 // Config represents the configuration state of a whisper node.
 type Config struct {
-	MaxMessageSize     uint32  `toml:",omitempty"`
-	MinimumAcceptedPOW float64 `toml:",omitempty"`
+	MaxMessageSize                        uint32  `toml:",omitempty"`
+	MinimumAcceptedPOW                    float64 `toml:",omitempty"`
+	RestrictConnectionBetweenLightClients bool    `toml:",omitempty"`
```

Comment:
How about the shorter `acceptLightClientConnection` or `connectToLightClient` ?

Reply:
Yes, it's shorter but I think it's less obvious.